play-json/shared/src/test/scala/play/api/libs/json/JsonSharedSpec.scala

```diff
@@ -133,10 +134,10 @@ class JsonSharedSpec extends AnyWordSpec with Matchers with org.scalatestplus.sc
       val writes: Writes[User]   = Json.writes[User]
       val owrites: OWrites[User] = Json.writes[User]
 
-      js.toJsObject(peach)(owrites) mustBe an[JsObject]
+      Predef.assert(js.toJsObject(peach)(owrites).isInstanceOf[JsObject])
```

Comment:
?

Reply:
I can't remember what piece, but I think it's `mustBe` that is a macro in ScalaTest, and we can't use ScalaTest's macros as they aren't expanded when this is compiled with Scala 3.  Similar to `illTyped`.